src/components/funFactsModal.vue

```diff
@@ -44,66 +46,64 @@ export default {
     return {
       currentIndex: 0,
       shuffledBlocks: this.shuffleBlocks(this.blocks)
-    };
+    }
   },
   methods: {
     close(e) {
-      if (e.target.className === "container") {
-        this.exit();
+      if (e.target.className === 'container') {
+        this.exit()
       }
     },
     exit() {
-      this.$emit('close');
+      this.$emit('close')
     },
     nextSlide() {
       if (this.currentIndex < this.shuffledBlocks.length - 1) {
-        this.currentIndex++;
+        this.currentIndex++
       } else {
-        this.currentIndex = 0;
+        this.currentIndex = 0
       }
     },
     prevSlide() {
       if (this.currentIndex > 0) {
-        this.currentIndex--;
+        this.currentIndex--
       } else {
-        this.currentIndex = this.shuffledBlocks.length - 1;
+        this.currentIndex = this.shuffledBlocks.length - 1
       }
     },
     shuffleBlocks(blocks) {
-      let shuffled = blocks.slice(); 
+      let shuffled = blocks.slice()
       for (let i = shuffled.length - 1; i > 0; i--) {
-        const j = Math.floor(Math.random() * (i + 1));
-        [shuffled[i], shuffled[j]] = [shuffled[j], shuffled[i]]; 
+        const j = Math.floor(Math.random() * (i + 1))
```

Comment:
this function looks like it was formatted a little strangely

Reply:
fixed